### ISSUE TITLE
Hosted file moved? 

### ISSUE THREAD

**Issue by yonghaoy** [igvteam/igv-notebook#19]
Hello, IGV.js is broken recently because `https://igv.genepattern.org/genomes/seq/hg38/hg38.fa.fai` is blocked by our Content Security Policy.
It is expected because we have Content Security Policy allowlist and that allowlist does not contain that url. 
IGV urls in our current allowlist are
```
    "https://s3.amazonaws.com/igv.broadinstitute.org/",
    "https://s3.amazonaws.com/igv.org.genomes/",
    "https://portals.broadinstitute.org/webservices/igv/",
    "https://igv.org/genomes/",
```

I am wondering if igv.js recently move their hosted files to igv.genepattern.org? 
I saw @jrobinso mentioned the moving here: https://github.com/igvteam/igv.js/issues/1570#issuecomment-1357951675
but I am not sure if that is the same thing. 

Thanks!

**Comment by yonghaoy** [comment]
Actually we are using igv-jupyter https://github.com/g2nb/igv-jupyter which wraps [igv.js](https://github.com/igvteam/igv.js) by way of [igv-notebook](https://github.com/igvteam/igv-notebook).

**Comment by jrobinso** [comment]
OK.   igv-jupyter is focused on the needs of the g2nb project, its not something I personally have any involvement in.   I don't understand how those url property value are working but maybe it uses some magic of some kind.   Anyway igv-jupyter project would be the place to discuss that.

**Comment by yonghaoy** [comment]
Hey @jrobinso need to reopen this issue again as we are broken by host move(and I believe that might not be an issue with igv-js or igv-notebook).
Now our content security policy complains about 
1: `https://igv-genepattern-org.s3.amazonaws.com/` URL not in our CSP allowlist
2: `https://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ncbiRefSeq.txt.gz` is not in our CSP allowlist.

I checked this reference https://s3.amazonaws.com/igv.org.genomes/genomes.json and did see those two urls are now used by hg38. 
And you also mentioned in another issue: https://github.com/igvteam/igv/issues/1306#issuecomment-1490518327 that you plan to refer to UCSC url directly. 
I want to confirm the breakage is expected, and we just need to add the two url in our CSP allowlist? 

Thanks

**Comment by jrobinso** [comment]
I don't understand question (1), but urls to data will change from time to time, these are not noted in change logs as they are not part of the application.   As I suggested above you should consider hosting these files yourselves, if you want absolute control,  we provide them as a service but it is not mandatory to use our (or UCSC's) hosted files.     That said these do not change often.

In the future we will be doing more direct references to UCSC hosted files, which will include the host you reference and possibly others in the UCSC domains.    I had already list https://igv-genepattern-org.s3.amazonaws.com/ earlier.

**Comment by yonghaoy** [comment]
Thanks @jrobinso .
For 1, we were just trying to understand why IGV breaks this time, and i want to confirm the breakage is caused by "IGV moves its hosted files (specified in that [genomes.json](https://s3.amazonaws.com/igv.org.genomes/genomes.json))".

For hosting our reference, we were thinking about that. 
For now, I got it working by using old reference url: 
```
import igv_notebook

igv_notebook.init()
igv_browser = igv_notebook.Browser(
    {
        "reference": {
            "id": "custom_hg38",
            "name": "Custom HG38 reference that works in Terra",
            "fastaURL": "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg38/hg38.fa",
            "indexURL": "https://s3.amazonaws.com/igv.broadinstitute.org/genomes/seq/hg38/hg38.fa.fai",
            "aliasURL": "https://s3.amazonaws.com/igv.org.genomes/hg38/hg38_alias.tab",
            "tracks": [
                {
                    "name": "Refseq Genes",
                    "format": "refgene",
                    "url": "https://s3.amazonaws.com/igv.org.genomes/hg38/ncbiRefSeq.sorted.txt.gz",
                    "indexed": "false",
                    "removable": "false",
                    "order": 1000000,
                    "infoURL": "https://www.ncbi.nlm.nih.gov/gene/?term=$$"
      }
            ]
        },
        "locus": "chr22:24,376,277-24,376,350"
    })

igv_browser.load_track(
    {
        "name": "1000004 CRAM",
        "url": "wgs_1000004.cram",
        "format": "cram",
        "type": "alignment",
        "indexURL": "wgs_1000004.cram.crai",
        "indexed": True
    })
igv_browser.show()
```

**Comment by jrobinso** [comment]
Yes that is what I recommend if you want full control over the URLs.  Specifically do not use the "genome: id" shortcut, but fully specify everything.
